Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -268,17 +298,32 @@ theorem stabilizer_vadd_eq_stabilizer_map_conj (g : G) (a : α) :
     neg_add_cancel, zero_vadd, ← mem_stabilizer_iff, AddSubgroup.mem_map_equiv,
     AddAut.conj_symm_apply]
 
+/-- The natural group equivalence between the stabilizers of two elements in the same orbit. -/
+def stabilizerEquivStabilizer {a b : α} {g : G} (hg : g +ᵥ b = a) :
```

Comment:
Does it work to get this one (and its theorems) by using `to_additive` on the earlier ones?

Reply:
No, because `AddAut.conj` is not exactly the translation of `MulAut.conj`, it also wraps things into an `Additive _`.